content/_code-samples/issue-a-token/js/list-account-tokens.js

```diff
@@ -0,0 +1,59 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Show tokens and other objects owned by an account.
+  // https://xrpl.org/account_objects.html#account_objects
+  // https://xrpl.org/account_lines.html#account_lines
+
+  async function main() {
+    // Testnet example: r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW
+    account = "r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW"
```

Comment:
```suggestion
if (typeof module !== "undefined") {
    // Use var here because const/let are block-scoped to the if statement.
    var xrpl = require('xrpl')
  }
  // Show tokens and other objects owned by an account.
  // https://xrpl.org/account_objects.html#account_objects
  // https://xrpl.org/account_lines.html#account_lines

  async function main() {
    // Testnet example: r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW
    account = "r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW"
```
The indentation looks off. Can you correct for the entire file?

Reply:
The indentation looks and seems to be correct on my computer

Reply:
Here's the indentation fixed - https://pastebin.com/E7tqsqha

Reply:
Thanks for pointing that out, I missed it (again)!